webhooks-extension/docs/GettingStarted.md

```diff
@@ -0,0 +1,161 @@
+
+Store the access token in a Kubernetes secret via the Tekton Dashboard's 'Secrets' menu. This panel will help you add the right annotation to the secret, and patch it onto the specified service account. Annotations should typically be of the form, `tekton.dev/git-0: https://github.com`. If you are using an access token, store it in the 'password' field. Be sure to specify the service account that will be used by the associated Tekton pipeline. 
+
+![Creating a Git secret](createGitSecret.png?raw=true "Creating a Git secret")
+
+### Create credentials: Docker
+
+Credentials are normally required to push images to a docker registry. Some docker registries support access tokens. Others such as docker hub only support username/password. If you are pushing images to Docker Hub the annotation should be of the form, `tekton.dev/docker-0: https://index.docker.io/v1/`. Again, select the service account of the associated Tekton pipeline so that it is correctly patched with the new docker credentials. 
+
+### Import pipeline definitions
+
+A Tekton Pipeline is composed of one or more Tasks. We expect that most users will store their Pipeline and Task definitions in git. You may wish to `git clone` and `kubectl apply` these definitions manually, or more likely, as part of an automated installation and setup. For your convenience, the dashboard provides a menu option, 'Import Tekton resources'. 
+
+![Import pipeline definitions](importPipelines.png?raw=true "Importing pipeline definitions")
+
+This screen shot shows a user importing pipeline definitions from a git repository. This panel will drive the Pipeline `pipeline0` 
+
+
+## Creating a new webhook
+
+The Tekton dashboard webhooks extension creates an association between a git repository and a Tekton pipeline. Changes to a branch or pull request in the associated git repository will cause a new PipelineRun and PipelineResource objects to be created. This section covers the various setup and configuration options in more detail. The image below shows the current form of the main 'Create Webhook' panel in the dashboard. 
+
+![Creating a new webhook](screenShot1.png?raw=true "Creating a new webhook")
+
+### Name
+
+A descriptive name for this webhook. Names must currently be unique; we aim to relax this restriction. 
+
+### Repository URL
+
+The URL of the git repository in which the webhook should be created. 
+
+### Access Token
+
+A GitHub access token that will be used to create the webhook. You may have created an access token in the section  'Create credentials: Git' above. If you chose to have one access token for both checking out code and creating webhooks then you can reuse its value here. 
+
+1. If you did not do so earlier, create an access token via GitHub. As of August 2019, for a given GitHub repository, visit [your name icon in the top right corner] > Settings > Developer settings > Personal access tokens > Generate new token and create a token with `admin:repo_hook` permissions. 
```

Comment:
I've read this before, we could condense this into a "Creating access tokens" reference we link to in both places

Reply:
I've used a link in the second case.